non-standard-plugins.js

```diff
@@ -4,6 +4,7 @@ module.exports = function shopifyNonStandardPlugins(options = {}) {
   const plugins = [
     require.resolve('@babel/plugin-proposal-class-properties'),
     require.resolve('@babel/plugin-syntax-dynamic-import'),
+    require.resolve('@babel/plugin-proposal-optional-catch-binding'),
```

Comment:
I think you need to add as a dependency?

Reply:
oh, weird. Not sure why my `package.json` change isn't in here 😕